README.md

```diff
@@ -14,3 +14,33 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+After each iteration, sortWidth doubles, so after x iterations, sortWidth will
+be $2^x$, and my outer while loop runs while $2^{x+1} < 2n$, as it must run so that
+it performs at least one iteration encompassing the entire input array.  
+Taking $\log_{2}$ of both sides of the inequality:  
+$\log_{2} 2^{x+1} < \log_{2} 2n$  
+$x + 1 < 1 + \log_{2} n$  
+$x < \log_{2} n$  
+So the outerloop runs in $\Theta(log n)$.  
+
+The for loop that iterates through subArrays will run $\frac{n}{sortWidth}$ times, or
+$\frac{n}{2}+\frac{n}{4}+\frac{n}{8}+...$, summing up to $n$, giving us $\Theta(n)$.  
+
+The next while loop, will run $O(m)$ times, where m is the size of the subArrays its
+merging, and when $subArrayMid \not\geq rightSortBound$, the for loop within may
+need to run a further $O(m)$ times, so when needing to shift m elements m times,
+we get $\Theta(n^2)$  
+
+Adding up these we get $\Theta(log_{2} n) + \Theta(n) +\Theta(n^2)$. As we don't
```

Comment:
Wait, can you just add those up? I thought the first was the loop iterations, so wouldn't you need to multiply something?